Examples/E3 - Cargo/Air.cs

```diff
@@ -0,0 +1,16 @@
+﻿
+namespace Examples.Cargo;
+
+public class Air : ITransport
+{
+    private static readonly Lazy<Air> _instance = new(() => new Air());
```

Comment:
it has Thread-Safe Overhead

Reply:
I don't fully understand your point. Do you mean I should not be concerned about multi-threading, or that managing multi-threading with Lazy<T> is not appropriate in this context? I've read about Lazy<T> recently, and I thought that because it is a built-in mechanism, it would have less overhead.

Reply:
> it has Thread-Safe Overhead

I don't fully understand your point. Do you mean I should not be concerned about multi-threading, or that managing multi-threading with Lazy<T> is not appropriate in this context? I've read about Lazy<T> recently, and I thought that because it is a built-in mechanism, it would have less overhead.